Lib/inspect.py

```diff
@@ -2560,7 +2560,7 @@ class BoundArguments:
 
     Has the following public attributes:
 
-    * arguments : OrderedDict
+    * arguments : Dict
```

Comment:
Should be: dict

(Docs don’t use English capitalization rules for code references like class names)